Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+load './lib/tasks/check_about_yml.rake'
```

Comment:
Because when it didn't work with 'require' you came over and tried 'load'
and it worked?

On Wed, Dec 2, 2015 at 10:57 AM, Mike Bland <notifications@github.com>
wrote:

> In Rakefile
> <https://github.com/18F/about_yml/pull/56#discussion_r46432009>:
>
> > @@ -1,5 +1,6 @@
> >  require 'bundler/gem_tasks'
> >  require 'rake/testtask'
> > +load './lib/tasks/check_about_yml.rake'
>
> Out of curiosity, why load and not require?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/about_yml/pull/56/files#r46432009>.
>